src/components/QuickGrid/QuickGrid.tsx

```diff
@@ -273,19 +237,20 @@ export class QuickGridInner extends React.Component<IQuickGridProps, IQuickGridS
         );
     }
 
+    onActionItemClick = (option, rowIndex) => {
+        const { onActionSelected, actionIconName, actionItems } = this.props.gridActions;
```

Comment:
Nice to pull the function out 👍 , but do you still need **actionIconName**?

Reply:
You are correct, it's not needed. It was a case of quick C/P :)